cmd/terramate/cli/cli.go

```diff
@@ -319,27 +319,6 @@ func (c *cli) run() {
 
 	c.checkVersion()
 
-	if c.parsedArgs.Changed {
```

Comment:
Here we moved the logic for checking git closer to the actual features, although this was not necessarily wrong (change detection needs git)